sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md

```diff
@@ -198,28 +214,23 @@ configured a new Azure AD B2C tenant and registered a new application in it, and
 application to use the Spring annotations and classes to protect the web app.
 
 ## Troubleshooting
-If you encounter any bug, please file an issue [here](https://github.com/Azure/azure-sdk-for-java/issues).
-
-To suggest a new feature or changes that could be made, file an issue the same way you would for a bug.
-
-You can participate community driven [![Gitter](https://badges.gitter.im/Microsoft/spring-on-azure.svg)](https://gitter.im/Microsoft/spring-on-azure)
+### Enable client logging
```

Comment:
Since we barely use the Java SDk in this starter, maybe we should add a section talking about how to change the Spring logging level?